<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDAlarmConditionGeneralDialog</class>
 <widget class="QDialog" name="CMDAlarmConditionGeneralDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="levelEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectLevelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_latched">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="需要人为确认后报警才会消失"/>
   </property>
   <property name="text">
    <string>报警确认</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_acknowledgeRequired">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="需要报警知悉操作"/>
   </property>
   <property name="text">
    <string>报警知悉</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectExpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_delayed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>延时(ms)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delayedEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="delayConditionEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>延时条件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDelayConditionExpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDelayTimeExpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="serveritySpinBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>220</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_enabled">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="需要人为确认后报警才会消失"/>
   </property>
   <property name="text">
    <string>使能</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enableAlarmEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectEnableAlarmExpButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
